CorsixTH/Lua/app.lua

```diff
@@ -313,6 +313,39 @@ function App:init()
   return true
 end
 
+---
+-- Initialises a Lua DBGp client and
+-- connects it to a DBGp server so that
+-- it can debug lua scripts at runtime.
+-- @return error_message
```

Comment:
I personally think the LDocGen function comment convention should be changed to be the same as the LuaDoc convention I've used here because a) Eclipse makes these multi-line comments blue where as all other Lua comments are green and b) a CorsixTH programmer could start a new comment line by pressing return without having to type: --!